plugins/org.yakindu.sct.doc.user/src/user-guide/lab_projects.textile

```diff
@@ -0,0 +1,77 @@
+
+h3(#lab_projects_typescript_codegen_blogs). Blog Articles
+
+"Typescript Code Generation With YAKINDU Statechart Tools":https://blogs.itemis.com/en/typescript-code-generation-with-yakindu-statechart-tools
+
+h3(#lab_projects_typescript_codegen_examples). Examples
+
+"Web-based Car HMI":https://www.itemis.com/en/yakindu/state-machine/documentation/examples/org-yakindu-sct-examples-ycar
+
+h2(#lab_projects_swift_codegen). Swift Code Generator
+
+The swift code generator was built as part of a student project. It is in a beta phase. Feel free to try out our example.
+
+h3(#lab_projects_swift_codegen_examples). Examples
+
+"Swift Showcase App":https://www.itemis.com/en/yakindu/state-machine/documentation/examples/org-yakindu-sct-examples-swift
+
+h2(#lab_projects_csharp_codegen). C# Code Generator
+
+The C# code generator is an external contribution and therefore open-source. Currently we have no resources on the C# code generator but we are working on providing a simple example. See How to install code generators for installation details.
+
+h2(#lab_projects_rkh_codegen). RKH Code Generator
+
+RKH (Reative frameworK for Hierarchical state machines) is a state machine framework used to build software for reactive and real-time embedded systems in a safely and timely way. In a cooperation with its developers at "VortexMakes":https://github.com/vortexmakes we created created a code generator which directly generates RKH compliant code out of YAKINDU statecharts. Read our blog articles and try out our examples if you want to know more:
+
+h3(#lab_projects_rkh_codegen_blogs). Blog Articles
+
+"Introduced: New RKH State Machine Framework for Reactive and Real-Time Embedded Systems":https://blogs.itemis.com/en/rkh-state-machine-framework-for-reactive-and-real-time-embedded-systems
+
+"Embedded System Development With YAKINDU Statechart Tools and RKH":https://blogs.itemis.com/en/embedded-system-development-with-yakindu-statechart-tools-and-rkh
+
+h3(#lab_projects_rkh_codegen_examples). Examples
+
+"Blinky LED for RKH Framework on STM32":https://www.itemis.com/en/yakindu/state-machine/documentation/examples/org-yakindu-sct-examples-rkh-blinky-stm32
+
+"Blinky LED for RKH Framework on Linux":https://www.itemis.com/en/yakindu/state-machine/documentation/examples/org-yakindu-sct-examples-rkh-blinky
+
+h2(#lab_projects_howto_install_codegen). How to Install More Code Generators
+
+The code generators listed in this chapter are not part of our official product bundles as their are currently in a beta phase. This means that you need to install them separately if you want to use them. The easiest way is to use the generator model wizard and select _Install More…_ This will list all available code generators that are not yet installed.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/gpINwwyfnqk"></iframe>
```

Comment:
Mmh, use css maybe with relative measurements, or mobile will hate you

Reply:
I checked with the other embedded videos on example page and they look fine also on mobile with the same settings.